### PR TITLE
Adds find_by_email to Subscriber and according specs.

### PR DESCRIPTION
**Pull request by ji**

### CODE REVIEW

File: rspreedly.gemspec [hunk 1]
```diff
@@ -32,5 +32,7 @@ Gem::Specification.new do |s|
     s.add_dependency(%q<httparty>, [">= 0.5.0"])
     s.add_dependency(%q<rspec>, [">= 0"])
     s.add_dependency(%q<webmock>, [">= 0"])
+    s.add_dependency(%q<pry>)
+    s.add_dependency(%q<pry-debugger>)
   end
```
Comment (rlivsey) [inline rspreedly.gemspec:37]: Can you make these development dependencies?

**Comment by ji** [comment]
Done! Sorry :).

**Comment by rlivsey** [comment]
Great thanks, do you want me to add you as a committer so you can push directly in future?

**Comment by ji** [comment]
Thanks for merging in. Committer access would be good. I'm working with rspeeesly a lot in the moment and might need some more additions to it, thanks :).

**Pull request merged**